app/src/main/java/com/github/sdpcoachme/map/MapActivity.kt

```diff
@@ -170,5 +212,74 @@ fun Map(modifier: Modifier, lastUserLocation: MutableState<LatLng?>) {
                 )
             }
         }
+
+        coachesToDisplay.map { user ->
+            val state by remember {
+                mutableStateOf(
+                    MarkerState(LatLng(user.location.latitude, user.location.longitude))
+                )
+            }
+
+            MarkerInfoWindowContent(
+                state = state,
+                tag = MARKER(user),
+                onInfoWindowClick = {
+                    // TODO: code similar to CoachesList, might be able to modularize
```

Comment:
We should definitely take a look at that code part in the next sprint to refactor common code parts. But for now, I think it is fine.